### PR TITLE
[docs] Updated runtests instructions

### PR DESCRIPTION
**Pull request by Aryamanz29**
- Updated `Runtests` section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -251,16 +251,16 @@ You can run tests either with `geckodriver` (firefox) or `chromedriver` (chromiu
 
    - Setup chromedriver
 
-     1. Install: `sudo apt --yes install chromium`
+     1. Install: `sudo apt --yes install chromium-browser`
```
Comment (Aryamanz29) [inline README.md:254]: - Running `sudo apt --yes install chromium` will produce 'Package' is not available.
```bash
sudo apt --yes install chromium        
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package chromium is not available, but is referred to by another package.
```

Comment (Aryamanz29) [reply README.md:254]: We need to update this in  `openwisp-utils`, `openwisp-controller`, `openwisp-monitoring` as well.

Comment (Aryamanz29) [reply README.md:254]: For **Ubuntu**, it is `chromium-browser` & for **Debian**, it's `chromium`.
Ref : https://linuxhint.com/install-chromium-ubuntu/
https://www.how2shout.com/linux/install-chromium-browser-debian-11-bullseye/

**Comment by Aryamanz29** [comment]
Rebase done :+1:

**Review by atb00ker — APPROVED** [review]
Looks good, thanks! 😄

**Pull request merged**